lib/cloudantBaseService.ts

```diff
@@ -142,6 +149,11 @@ export default abstract class CloudantBaseService extends BaseService {
     // Read external configuration and set as request defaults.
     super.configureService(serviceName);
     this.configureSessionAuthenticator();
+    // Add response interceptor for error transforms
+    this.getHttpClient().interceptors.response.use(
+      (response) => response,
+      (axiosError) => errorResponseInterceptor(axiosError)
+    );
```

Comment:
`configureService` overwrites the initial interceptor settings with `super.configureService(serviceName)` loosing the interceptor we have initially added in the constructor. Similarly to the session authenticator config this needs to be called here as well. 

The test case of `test_added_via_configureService` simulates what's happening when `new CloudantV1` then `configureService` happens (as in `newInstance`) so we can keep that test case, I think, but I can add another test where we call `newInstance` instead of `new Cloudant` if you insist.

Reply:
Oh I see it now via the fact that the `configureService` creates a new request wrapper with a new client.
Probably the test is fine, having an extra `newInstance` as well might protect us against divergence of that implenentation from what is simulated in the existing test, but I think we test `newInstance` in other paths so we're probably OK.